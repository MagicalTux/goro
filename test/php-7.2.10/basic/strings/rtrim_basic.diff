--
     string(10) "!===Hello "
     
     -- Trim some non-white space characters from a string using a character range --
013- string(10) "0123456789"
013+ string(35) "0123456789ABCDEFGHIJKLMNOPQRSTUVWXY"
     
     -- Trim the ASCII control characters at the beginning of a string --
016- string(14) "Example string"
016+ string(16) "Example string
017+ "
     ===DONE===
