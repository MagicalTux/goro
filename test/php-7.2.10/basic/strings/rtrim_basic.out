*** Testing rtrim() : basic functionality ***

-- Trim string with all white space characters --
string(27) "---These are a few words---"

-- Trim non-whitespace from a string --
string(15) "!===Hello World"

-- Trim some non-white space characters from a string --
string(10) "!===Hello "

-- Trim some non-white space characters from a string using a character range --
string(35) "0123456789ABCDEFGHIJKLMNOPQRSTUVWXY"

-- Trim the ASCII control characters at the beginning of a string --
string(16) "Example string
"
===DONE===